charts/sscs-evidence-share/values.aat.template.yaml

```diff
@@ -1,3 +1,6 @@
 java:
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
+  environment:
+    PDF_SERVICE_BASE_URL: https://docmosis-development.platform.hmcts.net/rs/render
```

Comment:
shouldn't this be in values.yaml?

Reply:
Good idea. That way it only needs to be overridden in prod